add_sort/(SC-98)

youki2018 opened the pull request.

youki2018 committed:
1番完了(SC-98)

youki2018 committed:
2番分らない(SC-98)

youki2018 committed:
同期用commit/SC-98

Commit:
同期用2/SC-98

Commit:
同期用3/SC-98

youki2018 committed:
エラーで動かない/SC-98

youki2018 committed:
ソートを追加 完了/SC-98

BMuscle commented on my_app/src/components/NoteSort.vue:
```diff
@@ -0,0 +1,26 @@
+  name: 'NoteSort',
+  data() {
+    return {
+      Sort: 'asc'
```
規約的にローワーキャメルケースを利用したいので、sortが良いと思います。

youki2018 replied:
Sortからsortに変更しました

youki2018 committed:
undefined　>< /SC-98

BMuscle commented on my_app/src/components/NoteSort.vue:
```diff
@@ -0,0 +1,32 @@
+  name: 'NoteSort',
+  data() {
+    return {
+      Selected: 'inode',
```
Selectedも、ローワーキャメルケースで、selectedの方が良いです。

youki2018 replied:
Selectedからselectedに変更しました

youki2018 committed:
更新日時のソートに使う更新日時が書かれた場所がわからない(SC-98)

できたこと
filterdNotes.vue内の変数をローワーキャメルケースに変更とは2つの値を受け渡しができるようになりソートを切り替えられるようになった

まだ＆わからないところ
更新日時が格納されている場所がわからず、ソートができない
StudyBook 開発ノート の クライアントDBのページを見たが変わらず…

youki2018 committed:
ソートの更新日時を追加(SC-98)

updated_atを指定した

youki2018 committed:
developの変更点を取り込んだ/(SC-98)

developから変更点を取り込んだ
